docs/asyncapi.yaml

```diff
@@ -37,171 +39,120 @@ components:
                 maximum: 9999999999
                 example: 1234567890
               apiLink:
-                type: url
                 description: api link for calling file stream of statement file that has been published unto blob
                 example: 'https://myStatementRetreivalApiEndpoint/statement-receiver/statement/v1/FFC_PaymentStatement_SFI_2022_1234567890_2022090615023001.pdf'  
               documentType:
                 type: string
                 enum:
                   - Payment statement
+                  - Payment schedule
                 description: document type
-                example: 'Payment statement'
+                example: 
+                  items:
+                    - Payment statement
+                    - Payment schedule
               scheme:
                 type: string
                 enum:
                   - SFI
```

Comment:
Again, seems incomplete

Reply:
Not sure what you mean here - could you elaborate? What do you believe to be missing?

Reply:
This suggests there's only 1 valid value for `Scheme` being the `enum` value `SFI`

Let's get a complete list of the possible schemes we support

Reply:
Where can I find the possible schemes support? Is it somewhere in ffc-pay-statement-generator?

Reply:
I'm gonna ask John to see if he knows :)

Reply:
John said SFI is the only one supported atm, so I'm going to leave this as it is